tools/c7n_gcp/c7n_gcp/resources/compute.py

```diff
@@ -53,6 +53,46 @@ def get_label_params(resource, all_labels):
                     }}
 
 
+@Instance.filter_registry.register('service-accounts')
+class ServiceAccounts(ValueFilter):
+    """Filters an instance by its authorized service accounts and their available list of scopes.
+
+    :example:
+
+    Filter all instances that grant the service account test123@developer.gserviceaccount.com
```

Comment:
`Filter all instances that grant the service account test123@developer.gserviceaccount.com` -> `Filter the service accounts (test123@developer.gserviceaccount.com) that grant full access to all Cloud APIs` ?